main.js

```diff
@@ -1,7 +1,28 @@
 import { courses } from './courses.js'
 
 const rows = courses.map(course => {
-  return `<tr><td>${course.CRSE} - ${course.DESCR}</td></tr>`
+  return `<tr><td>${course.CRSE} - ${course.DESCR} - ${course.ENROLLED}</td></tr>`
```

Comment:
do we want to show this? because i could see 3-4 people doing this with their own prs and the page looking clogged. feel free to disagree

Reply:
I am planning on only having mine show when the filter is clicked